lib/FindTests.js

```diff
@@ -74,29 +74,27 @@ function resolveCliArgGlob(
       ? globRelativeToProjectRoot.replace(/\\/g, '/')
       : globRelativeToProjectRoot;
 
-  return glob
-    .sync(pattern, {
-      cwd: projectRootDir,
-      nocase: true,
-      absolute: true,
-      ignore: ignoredDirsGlobs,
-      // Match directories as well and mark them with a trailing slash.
-      nodir: false,
-      mark: true,
-    })
-    .flatMap((filePath) =>
-      filePath.endsWith('/') ? findAllElmFilesInDir(filePath) : filePath
-    );
+  return globSync(pattern, {
+    cwd: projectRootDir,
+    caseSensitiveMatch: false,
+    absolute: true,
+    ignore: ignoredDirsGlobs,
+    // Match directories as well
+    onlyFiles: false,
+  }).flatMap((filePath) =>
+    // Directories have their path end with `/`
+    filePath.endsWith('/') ? findAllElmFilesInDir(filePath) : filePath
```

Comment:
Note to self: Does it end with a slash on Windows? We don’t seem to have test coverage for this condition being true.

Reply:
Added test case: [`36cdb47` (#647)](https://github.com/rtfeldman/node-test-runner/pull/647/commits/36cdb477d77236767181d7c4bb36086687defe19)

Btw, that test was failing on master – see https://github.com/rtfeldman/node-test-runner/pull/648. But it passed when installing glob 8 instead of 10. Another breakage by glob 🙈